pkg/serviceimport/map_test.go

```diff
@@ -84,6 +147,50 @@ var _ = Describe("ServiceImport Map", func() {
 				Expect(getIP(namespace1, service1)).To(Equal(secondIP))
 			}
 		})
+		When("we check for local cluster", func() {
+			When("cluster override IS NOT specified", func() {
+				It("should consistently return the relevant service IP per local cluster", func() {
+					for i := 0; i < 10; i++ {
+						Expect(getLocalClusterIP(namespace1, service1, "", clusterID1)).To(Equal(serviceIP1))
+						Expect(getLocalClusterIP(namespace1, service1, "", clusterID2)).To(Equal(serviceIP2))
+					}
+				})
+			})
+			When("cluster override IS NOT specified", func() {
+				It("should consistently return the IPs round-robin only for local cluster without service", func() {
+					ips := []string{serviceIP1, serviceIP2}
+					testRoundRobin(namespace1, service1, "", clusterID3, ips)
+					testRoundRobin(namespace1, service1, "", "", ips)
+				})
+			})
+			When("cluster override IS specified", func() {
```

Comment:
This think this is sufficiently covered above by the single cluster test cases above.